src/Gui/TaskView/TaskAppearance.cpp

```diff
@@ -97,7 +97,7 @@ void TaskAppearance::slotChangedObject(const Gui::ViewProvider& obj,
     // This method gets called if a property of any view provider is changed.
     // We pick out all the properties for which we need to update this dialog.
     std::vector<Gui::ViewProvider*> Provider = getSelection();
-    std::vector<Gui::ViewProvider*>::const_iterator vp = std::find_if
+    auto vp = std::find_if

```

Comment:
Lost `const`-ness.

Reply:
Would it not be more correct to const the definition of `Provider` on the previous line?

Reply:
Yes, agreed (assuming later code doesn't need to mutate... I didn't click through here to re-examine).

Reply:
done